addons/website/static/src/scss/website.scss

```diff
@@ -1246,6 +1257,59 @@ header {
     }
 }
 
+// Offcanvas menu on mobile or offcanvas themed header override
+.o_mobile_or_offcanvas {
```

Comment:
We have to find another name name for this element.
I'm sure that makes sense to you 'cause you worked in the task... but to me it's like `button_or_footer` ...